### PR TITLE
V5 pre-release

### PR DESCRIPTION
**Pull request by italoacasas**
The main reason about this PR is to update the `eslint` dependency to the last version `v3.5.0`.

In the process, I fix all the broken rules like `space-before-function-paren`, etc...

This known issues should be fixed:
- https://github.com/FlexShopper/eslint-config-flexshopper/issues/36
- https://github.com/FlexShopper/eslint-config-flexshopper/issues/37
- https://github.com/FlexShopper/eslint-config-flexshopper/issues/31
- https://github.com/FlexShopper/eslint-config-flexshopper/issues/30
- https://github.com/FlexShopper/eslint-config-flexshopper/issues/14

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,75 +1,57 @@
...
+        'indent': [ 'error', 4],
+        'linebreak-style': ['error', 'unix'],
+        'quotes': ['error', 'single'],
+        'semi': ['error', 'always'],
+        'max-len': ['error', 100],
+        'camelcase': ['error', { properties: 'never' }],
+        'eqeqeq': ['error', 'always'],
+        'strict': ['error', 'safe'],
+        'new-cap': ['error'],
+        'no-bitwise': ['error'],
```
Comment (johngills) [inline index.js]: @italoacasas Why not a warning on bitwise operators?

Comment (italoacasas) [inline index.js]: That rule is part of the ***yandex style guide***. if you feel like we should `warn` instead `break` we can adjust the rule.

Comment (johngills) [inline index.js]: I think we should warn.  What if you actually need a bitwise operator and it wasn't a typo?

Comment (italoacasas) [inline index.js]: done

File: index.js [hunk 2]
```diff
@@ -1,75 +1,57 @@
...
-        'no-proto': 2,
-        'no-script-url': 2,
-        'no-sequences': 2,
-        'no-undef': 2,
-        'no-shadow': 0,
+        'indent': [ 'error', 4],
+        'linebreak-style': ['error', 'unix'],
+        'quotes': ['error', 'single'],
+        'semi': ['error', 'always'],
+        'max-len': ['error', 100],
```
Comment (johngills) [inline index.js]: We are currently at 120. Can we keep it there?

**Pull request merged**